packages/sdk-js/src/index.ts

```diff
@@ -25,7 +22,6 @@ export {
   Attester,
   Verifier,
   BlockchainUtils,
-  SubscriptionPromise,
```

Comment:
Is this not needed anymore?

Reply:
Nope wasn't really needed in the first place, the before required interface lives in types.